docs/guides/modules/orchestrate/pages/scheduled-pipelines.adoc

```diff
@@ -14,35 +14,40 @@ Scheduled pipelines allow you to trigger pipelines periodically based on a sched
+Schedule triggers allow you to trigger pipelines periodically based on a schedule. Schedules can range from daily, weekly, monthly, or on a very specific timetable. To set up basic schedule triggers, you do not need any extra configuration in your `.circleci/config.yml` file. However, more advanced usage of the feature will require extra `.circleci/config.yml` configuration (for example, workflow filtering, or using parameters).
 
-Pipeline parameters are typed pipeline variables in the form of a string, integer, or boolean. Adding a parameter to a scheduled pipeline can be done in the web app in the triggers form while setting up a schedule. Any parameters set up in this manner must be added to your configuration file using the `parameters` key.
+Pipeline parameters are typed pipeline variables in the form of a string, integer, or boolean. Adding a parameter to a schedule trigger can be done in the web app in the triggers form while setting up a schedule. Any parameters set up in this manner must be added to your configuration file using the `parameters` key.
 
-Scheduled pipelines are set to run by an "actor", either the CircleCI scheduling system, or a specific user (for example, yourself). The scheduling actor is important to consider if making use of restricted contexts in workflows. If the user (actor) running the workflow does not have access to the context, the workflow will fail with the `Unauthorized` status.
+Schedule triggers are set to run by an "actor", either the CircleCI scheduling system, or a specific user (for example, yourself). The scheduling actor is important to consider if making use of restricted contexts in workflows. If the user (actor) running the workflow does not have access to the context, the workflow will fail with the `Unauthorized` status.
 
-You can find a basic how-to guide on the xref:set-a-nightly-scheduled-pipeline.adoc[Set a nightly scheduled pipeline] page, and more advanced examples on the xref:schedule-pipelines-with-multiple-workflows.adoc[Schedule pipelines with multiple workflows] pages.
+You can find a basic how-to guide on the xref:set-a-nightly-scheduled-pipeline.adoc[Set a nightly schedule trigger] page, and more advanced examples on the xref:schedule-pipelines-with-multiple-workflows.adoc[Schedule pipelines with multiple workflows] pages.
 
 [#get-started-with-scheduled-pipelines]
-== Get started with scheduled pipelines
+== Get started with schedule triggers
 
-To get started with scheduled pipelines, you have the option of using the API, or using the CircleCI web app. Both methods are described below.
+To get started with schedule triggers, you have the option of using the API, or using the CircleCI web app. Both methods are described below.
+
+// I think we need 3 tabs here
+// - Bitbucket Cloud using the web-app --> use the Triggers tab
+// - GitHub (OAuth and GitHub App) using the web-app --> use the Project Setup tab
+// - API (with caveat that it's not yet available for GitHub App) 
```

Comment:
I think we need to add tabs here. 
We have the content for Bitbucket & for API, but we're missing the one for GitHub App.